frontend/src/interface/projectInterface.js

```diff
@@ -44,39 +59,63 @@ export function getModels() {
  * Calculates the percentage values and returns them.
  * @param compositionData filtered data from backendconnnect
  */
-export function setMaterialCompositionData(compositionData) {
-    console.log('compositionData');
-    materialDataInPercent = [];
-    materialCompositionData = Array.from(compositionData);
-    console.log(materialCompositionData);
-    for (let i = 0; i < materialCompositionData.length; i++) {
-        materialDataInPercent[i] = Number(materialCompositionData[i]);
+export function setMaterialCompositionData(scenarioName, compositionData) {
+    if (scenarioName === scenarioNames.baseline) {
+        console.log('compositionData');
+        materialDataInPercent = [];
+        materialCompositionData = Array.from(compositionData);
+        console.log(materialCompositionData);
+        for (let i = 0; i < materialCompositionData.length; i++) {
+            materialDataInPercent[i] = Number(materialCompositionData[i]);
+        }
+        console.log(materialDataInPercent);
+    } else if (scenarioName === scenarioNames.modified) {
+        console.log('compositionData');
+        modifiedMaterialDataInPercent = [];
+        modifiedMaterialCompositionData = Array.from(compositionData);
+        console.log(modifiedMaterialCompositionData);
+        for (let i = 0; i < modifiedMaterialCompositionData.length; i++) {
+            modifiedMaterialDataInPercent[i] = Number(modifiedMaterialCompositionData[i]);
+        }
+        console.log(modifiedMaterialDataInPercent);
     }
-    console.log(materialDataInPercent);
 }
 
 /**
  * Getter method to recieve the filtered Material Composititon Data from API
  */
-export function getMaterialCompositionData() {
-    return materialDataInPercent;
+export function getMaterialCompositionData(scenarioName) {
+    if (scenarioName === scenarioNames.baseline) {
+        return materialDataInPercent;
+    } else if (scenarioName === scenarioNames.modified) {
+        return modifiedMaterialDataInPercent;
```

Comment:
but I guess this means that they just have to be put to display in a different place.

Reply:
yes. For now, the data is stored in two different sets of variables for Baseline and Modified scenarios. This data is called based on the scenario, if it is Modified scenario/Baseline Scenario.